dev/com.ibm.ws.transport.http/resources/OSGI-INF/l10n/metatype.properties

```diff
@@ -198,3 +198,9 @@ http.proxyredirect.httpsPort.desc=The (secure) port to redirect to. Incoming HTT
 
 http.autoDecompression=Auto decompression
 http.autoDecompression.desc=Specifies whether the HTTP Channel automatically decompresses incoming request body data.
+
+http.decompressionRatioLimit=Decompression ratio limit
+http.decompressionRatioLimit.desc=Specifies the maximum ratio of decompressed to compressed request body payload. This ratio is verified as the body is read by the HTTP channel, and if the decompressionTolerance is reached, the request will be terminated.
```

Comment:
Suggested change: "Specifies the maximum ratio of decompressed to compressed request body payload. This ratio is verified as the body is read by the HTTP channel, and if the decompressionTolerance is reached, the request is ended."

At minimum, change "is terminated" to "is ended."